MAUI/Circular-Charts/Plot-Area-Background-View.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: Plot Area Background View  in .NET MAUI Chart control | Syncfusion
+description: This section explains about how to configure the Plot Area Backgrund and its features in .NET MAUI Chart (SfCircularChart).
+platform: maui

```

Comment:
change title, file name and description like cartesian

Reply:
I have changed the file name and title, description content.